www/src/components/package-readme.js

```diff
@@ -40,9 +42,8 @@ const PackageReadMe = props => {
           }}
           href={githubUrl}
           aria-label={`${packageName} source`}
```

Comment:
Maybe remove this `aria-label` and replace it with an [`aria-labelledby` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) that links to the text `<span>View plugin on GitHub</span>`? 

cc @marcysutton is that best practice here?

Reply:
I'm pretty sure they do the same thing but it's better to not repeat yourself so 👍 for `aria-labelledby`

Reply:
Sure 👍